### PR TITLE
[RFR] better presentation for resource page

### PR DESCRIPTION
**Pull request by ThieryMichel**

### CODE REVIEW

**Comment by djhi** [comment]
screenshot ?

**Comment by ThieryMichel** [comment]
capture du 2017-02-08 16 02 32

File: src/app/js/resource/Resource.js [hunk 1]
```diff
@@ -26,22 +27,26 @@ export const ResourceComponent = ({ resource, datasetTitle, titleKey, loading, p
...
         return (
             <Card className="not-found">
-                <Link to="/home">
-                    <HomeIcon />
-                    {datasetTitle || polyglot.t('back_to_list')}
-                </Link>
-                <h1>{polyglot.t('not_found')}</h1>
+                <CardText>
+                    <Link to="/home">
+                        <HomeIcon />
```
Comment (djhi) [inline src/app/js/resource/Resource.js]: apply flex box and align-items to center the icon vertically

**Review by djhi — CHANGES_REQUESTED** [review]

File: src/app/js/resource/Resource.js [hunk 2]
```diff
@@ -26,22 +27,26 @@ export const ResourceComponent = ({ resource, datasetTitle, titleKey, loading, p
...
         <div>
             <Card>
-                <Link to="/home">
-                    <HomeIcon />
-                    {datasetTitle || polyglot.t('back_to_list')}
-                </Link>
-                <h1>{titleKey ? resource[titleKey] : resource.uri}</h1>
+                <CardText>
+                    <Link to="/home">
+                        <HomeIcon />
```
Comment (djhi) [inline src/app/js/resource/Resource.js]: same

**Review by djhi — CHANGES_REQUESTED** [review]

**Review by djhi — APPROVED** [review]

**Pull request merged**